src/Artsy/Router/Components/Boot.tsx

```diff
@@ -28,7 +28,15 @@ export interface BootProps {
   headTags?: JSX.Element[]
 }
 
-const { GlobalStyles } = injectGlobalStyles()
+const { GlobalStyles } = injectGlobalStyles(`
+  h1 {
+    font-style: inherit;
+    font-family: inherit;
+    font-weight: inherit;
+    font-size: inherit;
+    margin: 0;
+  }
```

Comment:
@eessex and @mdole - saw the convo earlier. When issues like those encountered with `h1` tags arise you can pass global styles in here. Ideally these then move over [to palette](https://github.com/artsy/palette/blob/master/src/helpers/injectGlobalStyles.tsx#L7).

Reply:
Excellent, glad to see this.